### ISSUE TITLE
Let contributors know why test are failing

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#9916]
Currently we show them a red x with no details of which tests failed.

**Comment by pdurbin** [comment]
> This is because https://jenkins.dataverse.org is behind a firewall.

The other day @donsizemore mentioned that when this server moves there's a chance we can open it up.

**Comment by pdurbin** [comment]
@DS-INRAE @bencomp @johannes-darms @julian-schneider @lubitchv and everyone else, there's a new PR from @srmanda-cs to check out!

- #12368

**Comment by pdurbin** [comment]
I just merged #12368!

Contributors, please try merging "develop" into your branch and tell me if you still can't tell why tests are failing! I hope you have enough permission. Also, we should probably follow up with some docs to explain where to look! 😅

**Comment by ashmanda-cs** [comment]
Please feel free to tag me directly here or on Zulip if the test failures are not showing up as intended

**Comment by pdurbin** [comment]
Yes! Here's the topic on Zulip we've been using: [#containers > goodbye-jenkins-you-did-an-amazing-job](https://dataverse.zulipchat.com/#narrow/channel/375812-containers/topic/goodbye-jenkins-you-did-an-amazing-job/with/597906930)
